liquid-base/src/GHC/List.spec

```diff
@@ -53,7 +53,7 @@ qualif LenSum(v:[a], xs:[b], ys:[c]): len([v]) = (len([xs]) - len([ys]))
 GHC.List.!!         :: xs:[a] -> {v: _ | ((0 <= v) && (v < len(xs)))} -> a
 
 
- zip :: xs : [a] -> ys:[b] 
+zip :: xs : [a] -> ys:[b] 
             -> {v : [(a, b)] | ((((len v) <= (len xs)) && ((len v) <= (len ys)))
             && (((len xs) = (len ys)) => ((len v) = (len xs))) )}
```

Comment:
Nice! I guess we don't have to indent the `&&` because its inside the `{ ... | ... }`

Reply:
Indentation is declaration-level, much like in Haskell.

The `zip` type signature must start at the same column as the other declarations in the block (that's the actual change on this line). The lines that belong to the `zip` type signature must be indented more than the first line of the declaration. But the actual indentation of lines 57 and 58 does not matter here, as long as it's more than that of line 56.

In Haskell, you can also do
```
x = 7
              + 10
      + 3
          + 5
```
and it is all fine, as long as none of these lines start on the first column (or even to the left of the `x` token).